web_01.html

```diff
@@ -158,7 +156,7 @@ <h1>Website Fonts</h1>
                         </div>
                         <div class="project__content">
                             <a href="#" class="project__image">
-                                <img src="/images/vectorized_ideas.jpg" alt="">    
+                                <img src="/images/vectorized_ideas.jpg"></img>    
```

Comment:
`img` tag doesn't have children and it is a self-closing tag similar to `<hr />`, `<link />` and etc hence it should look like this: `<img />`